.circleci/cimodel/data/binary_build_definitions.py

```diff
@@ -37,9 +37,10 @@ def gen_docker_image(self):
         docker_distro_prefix = miniutils.override(self.pydistro, docker_word_substitution)
 
         # The cpu nightlies are built on the pytorch/manylinux-cuda102 docker image
-        alt_docker_suffix = self.cuda_version or "102"
-        docker_distro_suffix = "" if self.pydistro == "conda" else alt_docker_suffix
-        return miniutils.quote("pytorch/" + docker_distro_prefix + "-cuda" + docker_distro_suffix)
+        alt_docker_suffix = self.gpu_version or "cuda102"
+        docker_distro_suffix = alt_docker_suffix if self.pydistro != "conda" else (
+            "cuda" if alt_docker_suffix.startswith("cuda") else "rocm")
+        return miniutils.quote("pytorch/" + docker_distro_prefix + "-" + docker_distro_suffix)
```

Comment:
@seemethere requested we have one ROCm docker repo `pytorch/manylinux-rocm` and use tags for versions.  Example, `pytorch/manylinux-rocm:3.7`.

Reply:
@seemethere this is actually something we should also probably do for cuda too? at least for the minor versions?